### DISCUSSION TITLE
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

### DISCUSSION THREAD

**Discussion by br-follow** [tiangolo/uvicorn-gunicorn-fastapi-docker#313] [Questions] [answered]
Attempting to run the docker container on a macbook with an M1 chip gives the following warning:

`The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

**Comment by DorskFR** [comment]
`FROM --platform=linux/amd64 tiangolo/uvicorn-gunicorn-fastapi:python3.9`

in your Dockerfile

or 

`platform: linux/amd64`

in your docker-compose

**Comment by flyingjoe** [comment]
This should be resolved once #118 is merged - we're using flyingjoe/uvicorn-gunicorn-fastapi:python3.9 in the meantime since the amd64 version was too slow.

**Comment by tedivm** [comment]
I opened #118, and in the meantime I've started publishing my own containers you can use.

* [uvicorn](https://github.com/multi-py/python-uvicorn)
* [gunicorn](https://github.com/multi-py/python-gunicorn)
* [gunicorn running uvicorn](https://github.com/multi-py/python-gunicorn-uvicorn)
* [oso](https://github.com/multi-py/python-oso)

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

And if you start from a base Python image you would get multi-arch by default. :sunglasses:
